.appveyor.yml

```diff
@@ -47,6 +48,9 @@ install:
   - set PATH=%PATH%;%GOPATH%\bin
   - go get -u github.com/golang/dep/cmd/dep
   - dep version
+  - appveyor DownloadFile https://dl.google.com/android/repository/sdk-tools-windows-3859397.zip
```

Comment:
Maybe use `ps: Start-FileDownload` like above for consistency? Also see the [docs](https://www.appveyor.com/docs/how-to/download-file/#start-filedownload-cmdlet).

Reply:
Done, the other command didn't work anyway, probably it was removed since the example I used was created.